src/tracks/playabletrack/wavetrack/ui/SpectrumCache.h

```diff
@@ -119,11 +118,11 @@ struct WaveClipSpectrumCache final : WaveClipListener
    void Invalidate() override; // NOFAIL-GUARANTEE
 
    /** Getting high-level data for screen display */
-   bool GetSpectrogram(const WaveClip &clip, SampleTrackCache &cache,
-                       const float *& spectrogram,
-                       const sampleCount *& where,
-                       size_t numPixels,
-                       double t0, double pixelsPerSecond);
+   bool GetSpectrogram(
+      const WaveClip& clip, const WideSampleSequence& sequence,
```

Comment:
Please comment that only the 0th channel of `sequence` is really used.

In the interim, this still works correctly for `WideSampleSequence` backed by a right channel track, which always ignores its partner.

Please comment in the PR that QA should test spectrograms of stereo tracks, in which there is some obvious difference of content between the channels, to be sure the correct channel's stuff is drawn in each rectangle.